main.js

```diff
@@ -3,46 +3,58 @@ const author = document.querySelector('.author');
 const form = document.querySelector('form');
 const displayBooks = document.querySelector('.container');
 const message = document.querySelector('span');
+
 // Is localStorage Empty
 if (localStorage.getItem('Book List') === null) {
   localStorage.setItem('Book List', JSON.stringify([]));
 }
+
 // Data Storage into localStorage
 const dataStore = JSON.parse(localStorage.getItem('Book List'));
 // localStorage data update
 function updateData() {
   localStorage.setItem('Book List', JSON.stringify(dataStore));
 }
-// Create data row
+
 function createBooks(items) {
   let books = '';
   for (let i = 0; i < items.length; i += 1) {
     books += `
-      <p>${items[i].title}</p>
-      <p>${items[i].author}</p>
-      <button onclick='removeitem(${i})'>Remove</button>
-      <hr />
-    `;
+      <div class="displayRow">
+        <ul>
+          <li>${items[i].title} by ${items[i].author}</li>
+        </ul>
+        <button onclick='removeitem(${i})'>Remove</button>
+      </div>
+      <hr/>
+      `;
   }
   return books;
 }
+
 // Fetching data from localStorage
 function fetchData() {
   displayBooks.innerHTML = `
-    <ul>
       ${createBooks(dataStore)}
-    </ul>
   `;
 }
-// Adding new data to localStorage
-function newData(bookTitle, bookAuthor) {
-  const Books = {
-    title: bookTitle,
-    author: bookAuthor,
-  };
-  dataStore.push(Books);
-  updateData();
-  fetchData();
+
+// Adding new data to localStorage using
+class NewBooks {
+  constructor(BookTitle, BookAuthor) {
+    this.btitle = BookTitle;
+    this.bauthor = BookAuthor;
+  }
+
+  newData() {
+    const Books = {
+      title: this.btitle,
+      author: this.bauthor,
+    };
+    dataStore.push(Books);
+    updateData();
+    fetchData();
+  }
 }
 
 // Removing  data from localStorage
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=it%20inside%20classes.-,Create%20class%20methods%20to%20add%20and%20remove%20books.,-Maintain%20all%20of), Kindly add this `removeItem` function as a method to the `NewBook` class you created above